### DISCUSSION TITLE
Any way to update this automatically?

### DISCUSSION THREAD

**Discussion by ySergi** [mifi/lossless-cut#2005] [Q&A]
If there is no way to update this automatically, please, as a suggestion, add an autoupdater that downloads the ZIP and extracts it automatically to the existing location of the program.

**Comment by HolgerAusB** [comment]
agree, for windows. Alternatively or additionally  via winget

**Reply by ySergi** [reply]
I already suggested adding this app to winget but the developer just refused to do it. 🤷‍♂️

**Reply by HolgerAusB** [reply]
It is freeware. The devs have every right to refuse any nice-to-have features to focus on the core ones.
I am very grateful that this fantastic app continues to be maintained.

**Reply by ySergi** [reply]
Whether it is free or not, it does not exempt it from doing basic functionalities to improve the user's quality of life.

**Reply by mifi** [reply]
the app is open source. you're welcome to create a PR to add support for winget if you want to see support for that. auto updates are available on the [Windows Store version](https://apps.microsoft.com/detail/9p30lsr4705l?rtc=1&hl=nb-no&gl=NO) #1279
